strikt-core/src/main/kotlin/strikt/api/Assertion.kt

```diff
@@ -225,8 +225,14 @@ interface Assertion {
         )
         else -> {
           val fieldName = try {
-            val line = FilePeek.getCallerFileInfo().line
-            ParsedMapInstruction(line).body.trim()
+            val line = FilePeek(
+              listOf(
```

Comment:
this should probably not be created over and over again but I'm not sure where to put it.